services/wallet/router/pathprocessor/processor_swap_paraswap.go

```diff
@@ -34,11 +34,33 @@ type SwapParaswapProcessor struct {
 	priceRoute     sync.Map // [fromChainName-toChainName-fromTokenSymbol-toTokenSymbol, paraswap.Route]
 }
 
+const (
+	partnerID = "status.app"
+)
+
+func getPartnerAddressAndFeePcnt(chainID uint64) (common.Address, float64) {
+	const partnerFeePcnt = 0.7
+
+	switch chainID {
+	case walletCommon.EthereumMainnet:
+		return common.HexToAddress("0xd9abc564bfabefa88a6C2723d78124579600F568"), partnerFeePcnt
```

Comment:
@iurimatias please confirm this is not me trying to steal fee money

Reply:
"0xd9abc564bfabefa88a6C2723d78124579600F568" could also be constant that can we easily switched later?